Src/Microsoft.Dynamic/Utils/ReflectionUtils.cs

```diff
@@ -263,7 +263,7 @@ private static IEnumerable<T> WithBindingFlags<T>(this IEnumerable<T> members, F
         //
         // The general algorithm that is used for determining the names in a type and the layout of objects of the type is roughly as follows:
         // - Flatten the inherited names (using the hide by name or hide by name-and-signature rule) ignoring accessibility rules. 
-        // - For each new member that is marked �expect existing slot�, look to see if an exact match on kind (i.e., field or method), 
```

Comment:
This should not be changed.

Reply:
Maybe my VS2017 code format destroyed some special characters...
Let me try to restore it...

Reply:
The problem comes from my Git Bash for Windows. It clones the repo and converts the encoding of the file from UTF8 to ASCII based local code page.